pyproject.toml

```diff
@@ -34,7 +34,7 @@ line_length = 120
 [tool.coverage.run]
 branch = true
 source = ["."]
-omit = ["**/tests/**.py"]
+omit = ["**/tests/*.py"]
```

Comment:
In other repositories we are using `omit = ["**/tests/**"]`, for instance: https://github.com/moneymeets/youtrack-sdk/blob/92106b45ffcaae8eb967a742782ae31414bbae61/pyproject.toml#L38

Reply:
For updated version of coverage 
omit = ["**/tests/**"] will fail (not work) I can confirm that

There was major changes to file pattern matching
reference: https://coverage.readthedocs.io/en/stable/changes.html#version-7-0-0b1-2022-12-03

Reply:
Okay, so you mean we need to update https://github.com/moneymeets/youtrack-sdk/blob/master/pyproject.toml#L38 also? This is already merged with latest version.